DOC_CHANGES.md

```diff
@@ -7,3 +7,69 @@ Description of the required change.
 -->
 
 # Ohai Doc Changes:
+
+## Shard Seed Plugin
+
+The new `shard_seed` attribute provides a normally distributed integer ID value
```

Comment:
Hopefully it's not an approximately normal distribution.  I think md5 would be close to a uniform distribution though.
http://michiel.buddingh.eu/distribution-of-hash-values